tcf_website/templates/course/course_professor.html

```diff
@@ -193,7 +193,7 @@ <h2 class="mr-md-3">{{ course.title }}</h2>
 
         <div class="reviews-container mb-5 px-3">
             <div class="reviews-toolbar pb-2 d-flex flex-md-row flex-column justify-content-left justify-content-md-center">
-                <h3 class="reviews-heading mb-0 mt-1 mr-4">{{ reviews|length }} Comments</h3>
+                <h3 class="reviews-heading mb-0 mt-1 mr-4">{{ reviews|length }} {% if reviews|length == 1 %} Comment {% else %} Comments {% endif %}</h3>
```

Comment:
Just in general since we're eventually moving to be able to comment on reviews, it might be best to change Comments to Reviews?

Reply:
Oh wait I see, this displays total comments (reviews and comments) whereas the above displays number of reviews right?